### PR TITLE
feat(user): added generate/register Linnia user functionality

### PR DESCRIPTION
**Pull request by daniel-lanciana**
Should be able to generate/register without going to: consensys.github.io/linnia-faucet

### CODE REVIEW

File: src/components/GetUser.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      dataHash: '',
+      privateKey: '',
+    };
+
+    // Set variables pass as url arguments
+    window.location.search.substr(1).split('&').forEach((param) => {
```
Comment (MaxBlaushild) [inline src/components/GetUser.js]: You don't need this here. Please remove

File: src/components/GetUser.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  text: {
+    marginTop: 20,
+  },
+};
+
+class GetUser extends Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
```
Comment (MaxBlaushild) [inline src/components/GetUser.js]: You don't need these state properties. Please remove

Comment (daniel-lanciana) [reply src/components/GetUser.js]: Hi Max, copy/paste leftovers -- removed.

File: src/components/GetUser.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  text: {
+    marginTop: 20,
+  },
+};
+
+class GetUser extends Component {
+  constructor (props) {
```
Comment (MaxBlaushild) [inline src/components/GetUser.js]: Sorry missed this as well. The whole constructor can go. Last thing I promise :D

Comment (daniel-lanciana) [reply src/components/GetUser.js]: Np, happy to iterate as much as needed   :)

Comment (daniel-lanciana) [reply src/components/GetUser.js]: (done BTW)

**Pull request merged**